cargo-dist/src/init.rs

```diff
@@ -997,6 +1060,7 @@ fn apply_dist_to_metadata(metadata: &mut toml_edit::Item, meta: &DistMetadata) {
         bin_aliases: _,
         system_dependencies: _,
         github_build_setup: _,
+        mac_pkg_config: _,
```

Comment:
Wait doesn't this make init pointless.

Reply:
I left a comment above about having difficulty with the `init` stuff because it's a struct rather than two bare fields - that's why this is currently omitted. Was hoping to talk with you about it and/or make this depend on config 1.0, where I assume we'll have better handling.